spin_coater_pwm.c

```diff
@@ -0,0 +1,41 @@
+#include "spin_coater.h"
+#include "spin_coater_pwm.h"
+#include "hardware/clocks.h"
+#include "pico/divider.h"
+
+int
+pwm_set_freq_duty(uint32_t slice_num,
```

Comment:
We already use names `pwm_*` for heating/furnace PWM. Can we rename to `spinpwm_*` or something like that?

Reply:
done